static_src/stores/service_plan_store.js

```diff
@@ -62,6 +62,27 @@ class ServicePlanStore extends BaseStore {
         break;
       }
 
+      case serviceActionTypes.SERVICE_INSTANCES_RECEIVED: {
+        const instances = this.formatSplitResponse(action.serviceInstances);
+        this.fetched = false;
+        this.fetching = true;
+        this.emitChange();
+        const planRequests = [];
+        for (const instance of instances) {
+          planRequests.push(cfApi.fetchServicePlan(instance.service_plan_guid));
```

Comment:
I like the strategy of not fetching by default, as long as it is easy to override.

We could do something simple like:
``` 
planRequests = instances.filter((instance) => {
  return !this.get(instance.service_plan_guid);
}).map((instance) => {
  return cfApi.fetchServicePlan(instance.service_plan_guid);
});
```